src/components/trade/LastTrades/LastTradesWidget.tsx

```diff
@@ -23,11 +23,24 @@ const GET_TRADES = gql`
   }
 `
 
+interface TradeFromGraph {
```

Comment:
I was hoping to autogenerate this...

Reply:
Me too, maybe later. https://github.com/dotansimha/graphql-code-generator#readme is good